aggregator/src/aggregator.rs

```diff
@@ -590,17 +603,17 @@ impl<C: Clock> Aggregator<C> {
             .handle_delete_collection_job(&self.datastore, collection_job_id)
             .await?;
 
-        Ok(StatusCode::NO_CONTENT.into_response())
+        Ok(())
```

Comment:
Does this still return HTTP 204?

Reply:
Yes, I moved that to the end of the `collection_jobs_delete()` function.